tcf_website/templates/reviews/review.html

```diff
@@ -36,48 +40,69 @@
-                        <div class="ml-4 text-muted">
-                            <i class="far fa-smile-beam"></i>
-                            {{ review.enjoyability }}
-                        </div>
-                        <div class="ml-4 text-muted">
-                            <i class="fa fa-dumbbell fa-fw"></i>
-                            {{ review.difficulty }}
-                        </div>
-                        <div class="ml-4 text-muted">
-                            <i class="fa fa-hourglass-half fa-fw"></i>
-                            {{ review.hours_per_week }}
+                <div class="review-content">
+                  <p class="card-text mb-2" style="white-space: pre-line;">{{ review.text }}</p>
+                </div>
+
+                <div class="review-footer row mt-3">
+                    <!-- Review Ratings -->
+                    <div class="review-ratings col-sm-8 d-flex align-items-end">
+                        <div class="d-flex text-center flex-wrap mb-2">
+                            <div id="review-average" class="font-weight-bold">
+                                <i class="fa fa-star fa-fw"></i>
+                                {{ review.average|floatformat:2 }}
+                            </div>
+                            <div class="ml-4 text-muted">
+                                <i class="fas fa-thumbs-up"></i>
+                                {{ review.recommendability }}
+                            </div>
+                            <div class="ml-4 text-muted">
+                                <i class="fa fa-user"></i>
+                                {{ review.instructor_rating }}
+                            </div>
+                            <div class="ml-4 text-muted">
+                                <i class="far fa-smile-beam"></i>
+                                {{ review.enjoyability }}
+                            </div>
+                            <div class="ml-4 text-muted">
+                                <i class="fa fa-dumbbell fa-fw"></i>
+                                {{ review.difficulty }}
+                            </div>
+                            <div class="ml-4 text-muted">
+                                <i class="fa fa-hourglass-half fa-fw"></i>
+                                {{ review.hours_per_week }}
+                            </div>
                         </div>
                     </div>
 
-                    <div class="text-right">
-                        <p>{{ review.semester }}</p>
-                        <p id="date">{{ review.created|date }}</p>
+                    <div class="review-date col-sm-4 pl-0 text-right text-tcf-orange">
+                        <h6 class="font-weight-bold">{{ review.semester }}</h6>
+                        <h6 id="date">Updated {{ review.created|date }}</h6>
                     </div>
+
                 </div>
             </div>
         </div>
     </div>
 </div>
+
+<script type="module">
```

Comment:
Did you move the script because of the file structure?

Reply:
I thought the script is "inherited" from `reviews.html`

Reply:
Oh does it prevent having to import it in `user_reviews.html` as well? That makes sense with separation of concerns
Does it affect performance though? (since every review on a page has the script)

Reply:
^ooh that's true, didn't think of that. I'll check it out on silk

Reply:
Also not sure how Django builds work but maybe the js is optimized/minified in some way (don't think so though)

Reply:
I tested with django-silk and there wasn't really a difference in performance, however it looks like the move broke the responsiveness of the voting so I'll just put this script back in `reviews.html` lol